cibuildwheel/options.py

```diff
@@ -265,10 +265,12 @@ class EnvironmentFormat(OptionFormat):
     values may contain variables or command substitutions.
     """
 
-    def format_table(self, table: SettingTable) -> str:
+    @staticmethod
+    def format_table(table: SettingTable) -> str:
         return " ".join(f'{k}="{v}"' for k, v in table.items())
 
-    def merge_values(self, before: str, after: str) -> str:
+    @staticmethod
+    def merge_values(before: str, after: str) -> str:
```

Comment:
These 2 changes shall probably be reverted, they do not respect the prototype inherited from OptionFormat

Reply:
Yes, they do (in fact, mypy would complain if they didn't!). A `staticmethod` or `classmethod` is a perfectly valid implementation of an instance method. A `staticmethod` can also be an implementation of a `classmethod`. It's the other way around that's not allowed. I can't make the `OptionFormat` prototype a `staticmethod`, because then I can't ever make it anything else.

Reply:
Sadly, pylint is tripping up on this, because it's doing simple argument counting, and not taking into account that the `staticmethod` decorator adds the missing argument for you.

Reply:
I reverted it to make the pylint warning go away.  Though I think W0221 is incorrect and should take into account `@staticmethod`.